src/keys/CompositeKey.cpp

```diff
@@ -64,10 +67,14 @@ CompositeKey& CompositeKey::operator=(const CompositeKey& key)
 
     clear();
 
-    for (const Key* subKey : asConst(key.m_keys)) {
+    Q_FOREACH (const Key* subKey, key.m_keys) {
```

Comment:
Alright. But don't forget to use qAsConst() to avoid detaching and unwanted deep copies.